src/components/Autocomplete/components/ComboBox/components/Context/Context.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import {ComboBoxContext} from '../../../types';
+
+const {Provider, Consumer} = React.createContext<ComboBoxContext>({});
```

Comment:
If `useContext` only accepts `createContext` I can make a quick change once all the context's are change, and agree it would go better in `{Component}/Context.tsx`

Reply:
(nitpick: should that be `{Component}/context.tsx` as capital letters are for files that export a react component?)

I'm happy to do everything in this style for now and then we change over to the proposed new style all in one go :)

Reply:
Resolving this as we can make this change, along with an equivalent change for all usages of context in a follow up PR